dc-cudami-editor/src/components/modals/IframeAdderModal.jsx

```diff
@@ -42,36 +42,42 @@ class IframeAdderModal extends Component {
           {t('insert.iframe')}
         </ModalHeader>
         <ModalBody>
-          <FormGroup>
-            <Input
-              onChange={evt => this.setState({src: evt.target.value})}
-              placeholder="URL"
-              type="text"
-              value={this.state.src}
-            />
-          </FormGroup>
-          <FormGroup>
-            <Input
-              onChange={evt => this.setState({width: evt.target.value})}
-              placeholder={t('width')}
-              type="text"
-              value={this.state.width}
-            />
-          </FormGroup>
-          <FormGroup className="mb-0">
-            <Input
-              onChange={evt => this.setState({height: evt.target.value})}
-              placeholder={t('height')}
-              type="text"
-              value={this.state.height}
-            />
-          </FormGroup>
+          <Form
+            onSubmit={evt => {
+              evt.preventDefault()
+              this.addIframeToEditor()
+            }}
+          >
+            <FormGroup>
+              <Input
+                onChange={evt => this.setState({src: evt.target.value})}
+                placeholder="URL"
+                required
+                type="url"
+                value={this.state.src}
+              />
+            </FormGroup>
+            <FormGroup>
+              <Input
+                onChange={evt => this.setState({width: evt.target.value})}
+                placeholder={t('width')}
+                type="text"
```

Comment:
Ist es Absicht dass Width/Height Strings sind? Falls es möglich ist Suffixe wie `px`, `em`, `%` anzugeben, wäre es gut das noch irgendwo in der UI zu indizieren, ansonsten ist die Erwartung bei `width`/`height` glaube ich dass es um Integer geht.

Reply:
Ja, diese Suffixe sind auch möglich. Wie würdest du das in der UI kenntlich machen?

Reply:
- Im Placeholder String ein Beispiel mit `%` oder `px` machen
- Im Beschreibungs-String darauf hinweisen dass es CSS Syntax ist

Reply:
Beschreibungstext ist eingebaut.